src/datasources/wallets/entities/wallets.entity.db.ts

```diff
@@ -19,6 +19,7 @@ export class Wallet implements z.infer<typeof WalletSchema> {
 
   @ManyToOne(() => User, (user: User) => user.id, {
     onDelete: 'CASCADE',
+    nullable: false,
```

Comment:
This otherwise defaults to `true` meaning that `Wallet`s could be added without a `User`.